markdown/ch6.markdown

```diff
@@ -661,9 +660,9 @@ Ls = Ds - Cs
 ```
 
 Now we want to get the processing depth. Remember that getting the
-pseudo-inverse of the Laplacian matrix is out of the question, because it will
+pseudoinverse of the Laplacian matrix is out of the question, because it will
```

Comment:
I see all sorts of spellings out there: Golub & Van Loan use "pseudo-inverse", and wikipedia alternates between pseudoinverse and pseudo inverse.  I cannot definitively determine the correct one.  Any thoughts, Ross?

Reply:
No, my only thought was to be consistent within the chapter - these were the only two instances of "pseudo-inverse" whereas the other three instances were all "pseudoinverse". I went with the non-hyphenated spelling because it's what I encountered first.

Maybe it's fine to leave both spellings as it seems to reflect real ambiguity in the wider spelling of the term.

Reply:
No, consistency is great, thanks!